doc/Overview/Mvux/Advanced/Pagination.md

```diff
@@ -0,0 +1,398 @@
+        (int)Math.Ceiling(GetPeople().Length / (double)pageSize);
+
+    private Person[] GetPeople() =>
+        new Person[]
+        {
+            new("Liam", "Wilson"),
+            new("Emma", "Murphy"),
+            new("Noah", "Jones"),
+            new("Olivia", "Harris"),
+            new("William", "Jackson"),
+            /* ... more names ... */
+        }
+    }
+}
+```
+
+> [!NOTE]
+> You will find unsigned integers (`uint`) used in the context of Pagination as unsigned integeners do not allow negative values and prevent them in the first place.
+
+### Model
+
+```csharp
+using Uno.Extensions.Reactive;
+
+namespace PaginationPeopleApp;
+
+public partial record PeopleModel(IPeopleService PeopleService)
+{
+    const uint PageSize = 20;
+
+    public IListFeed<Person> PeopleAuto =>
+        ListFeed.AsyncPaginated(async (PageRequest pageRequest, CancellationToken ct) =>
+            await PeopleService.GetPeopleAsync(pageSize: PageSize, pageIndex: pageRequest.Index, ct));
+}
+```
+
+The `AsyncPaginated` method generates a `ListFeed` that supports pagination.
+
+Whenever the user scrolls down to see additional data and is hitting the end of the collection displayed in a `ListView`, the pagination List-Feed is automatically triggered with a page request.  
+The parameter of `AsyncPaginated`, is a delegate taking in a [`PageRequest`](#the-pagerequest-type) value and a `CancellationToken` and returning an `IListFeed<T>` where `T` is `Person` in our case. This delegate is invoked when a page request comes in.
+
+As you can see, we are sending the `Index` property of the incoming `pageRequest` argument to determine what page we're positioned.  
+`PageSize` refers to a constant value of `20`, but the `PageRequest` also has a `DesiredSize` property which the `ListView` can set according to its capacity. So essentially you can substitute `pageSize: PageSize` with `pageSize: pageRequest.DesiredSize` which is a nullable `uint`, and is populated by the `ListView`. We used a constant `PageSize` for the sake of this demonstration.
+
+### View
+
+```xml
+<Page 
+    x:Class="PaginationPeopleApp.MainPage"
+	xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:PaginationPeopleApp"
+    xmlns:mvux="using:Uno.Extensions.Reactive.UI">
+
+    <Page.Resources>
+        <DataTemplate x:Key="PersonDataTemplate" x:DataType="local:Person">
+            <StackPanel Orientation="Horizontal" Spacing="2">
+                <TextBlock Text="{x:Bind FirstName}"/>
+                <TextBlock Text="{x:Bind LastName}"/>
+            </StackPanel>
+        </DataTemplate>
+    </Page.Resources>
+
+    <ListView ItemsSource="{Binding PeopleAuto}" ItemTemplate="{StaticResource PersonDataTemplate}"
+        DataFetchSize="20" Header="Next page auto-loading when on available capacity or scrolled"/>
```

Comment:
We should not set the `DataFetchSize`, I'm not even sure that is supported by Uno.